src/Serilog.Sinks.Trace/Sinks/Trace/TraceSink.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Serilog.Sinks.Trace;
 
-class TraceSink : ILogEventSink
+sealed class TraceSink : ILogEventSink
```

Comment:
Visual Studio is suggesting using a primary constructor here - are we dpong those or leaving that at the 'traditional' style?